lib/cdo/script_constants.rb

```diff
@@ -51,6 +51,7 @@ module ScriptConstants
       # you must also update the cdo-tutorials gsheet so the end of script API works
 
       nil,
+      MINECRAFT_AQUATIC_NAME = 'aquatic'.freeze,
```

Comment:
heads up that we will have a merge conflict on this line with #25522, but that's expected 👍